schedule/functional/allpatterns.yaml

```diff
@@ -48,7 +48,7 @@ schedule:
     - console/zypper_in
     - console/zypper_log
     - console/yast2_i
-    - console/yast2_bootloader
+    - yast2_cmd/yast2_bootloader
```

Comment:
You can remove it from functional group's schedules.

Reply:
Done